extension/rusheye/arquillian-graphene-rusheye-test/src/test/java/org/arquillian/graphene/rusheye/test/RichFacesTest.java

```diff
@@ -0,0 +1,177 @@
+package org.arquillian.graphene.rusheye.test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.arquillian.graphene.rusheye.comparator.OcularResult;
+import org.arquillian.graphene.rusheye.pages.RichFaces;
+import org.arquillian.graphene.rusheye.pages.RichFaces60PercentSimilarity;
+import org.arquillian.graphene.rusheye.pages.RichFacesPoll;
+import org.arquillian.graphene.rusheye.pages.RichFacesTheme;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.graphene.page.Page;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+@RunAsClient
+public class RichFacesTest extends RichFaceBaseTest {
+
+    @Page
+    private RichFaces richFacesPage;
+    
+    @Page
+    private RichFaces60PercentSimilarity richFacesPage60;
+    
+    @Page
+    private RichFacesPoll richFacesPoll;
+    
+    @Page
+    private RichFacesTheme richFacesTheme;
+    
+    
+    /*
+     * 
+     *                Page objects visual validation
+     * 
+     */
```

Comment:
why this huge comment - a single line is enough I think.  ...or splitting into several test classes